libs/vkd3d/swapchain.c

```diff
@@ -2402,6 +2651,11 @@ static HRESULT dxgi_vk_swap_chain_init(struct dxgi_vk_swap_chain *chain, IDXGIVk
     if (FAILED(hr = dxgi_vk_swap_chain_init_waiter_thread(chain)))
         goto err;
 
+    if (FAILED(hr = dxgi_vk_swap_chain_init_low_latency(chain)))
+        goto err;
+
+    d3d12_device_add_ref(queue->device);
```

Comment:
As mentioned earlier, the command queue is what we need to hold ref on, not device.

Reply:
Fixed, thanks for the clarification.